releasenotes/notes/remove-toqm-optional-extra-90e974b64ec4a3bd.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The ``toqm`` optional setuptools extra that previously enabled running
+    ``pip install qiskit-terra[topm]`` has been removed as nothing in the
+    Qiskit code base is currently using that package anymore. If you'd like
+    to use the qiskit TOQM transpiler plugin you should install the
+    ``qiskit-toqm`` package directly.
```

Comment:
the `bip-mapper` extra is also gone - was that intentional, and does it need a reno if so?

Reply:
I did the release note in: https://github.com/Qiskit/qiskit-terra/pull/10526 which is the larger removal of the pass.

Reply:
Ok, we'll need to remember to re-add it when we backport this to do the 0.25.1 deployment.